Modules/Quotas/test/Quotas.Application.Tests/Tests/Identities/ListQuotasForIdentity/HandlerTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Backbone.BuildingBlocks.Application.Abstractions.Infrastructure.UserContext;
+using Backbone.Modules.Quotas.Application.Identities.Queries.ListQuotasForIdentity;
+using Backbone.Modules.Quotas.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Quotas.Domain.Aggregates.Identities;
+using Backbone.Modules.Quotas.Domain.Aggregates.Metrics;
+using Backbone.Modules.Quotas.Domain.Aggregates.Tiers;
+using FakeItEasy;
+using FluentAssertions;
+using Xunit;
+using static Backbone.UnitTestTools.Data.TestDataGenerator;
+
+namespace Backbone.Modules.Quotas.Application.Tests.Tests.Identities.ListQuotasForIdentity;
+
+public class HandlerTests
```

Comment:
You should add a test where you have multiple quotas in a single group.
I would be okay with updating the current test as well.